composer.json

```diff
@@ -38,7 +38,8 @@
         "vanilla/htmlawed": "~2.0",
         "vanilla/legacy-oauth": "~1.0",
         "vanilla/legacy-passwords": "~1.0",
-        "vanilla/nbbc": "~2.1"
+        "vanilla/nbbc": "~2.1",
+        "vanilla/garden-schema": "~1.0"
```

Comment:
Always require the most recent version when adding a package and use the "^" semantic requirement.